test/WebJobs.Extensions.DurableTask.Tests.csproj

```diff
@@ -33,4 +33,8 @@
     <Compile Include="$(SolutionDir)\.stylecop\GlobalSuppressions.cs" Link="GlobalSuppressions.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Do you know what this is?

Reply:
I don't know. It is not my intent. When I search the internet <ItemGroup><Service/> ... element, I found it is a notation to tell vs that this project is test project with 3rd party library. 
https://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for

Shall we remove this or keep this?

Reply:
Since it wasn't there originally and you didn't add it intentionally, I would remove it.

Reply:
Removed!